package/boot/uboot-rockchip/patches/109-rockchip-add-FriendlyElec-NanoPi-R3S.patch

```diff
@@ -0,0 +1,666 @@
+--- /dev/null
```

Comment:
please renumber this patch to 100, i.e. `package/boot/uboot-rockchip/patches/100-rockchip-add-FriendlyElec-NanoPi-R3S.patch`